_posts/2023-07-17-trino-fest-2023-comcast-recap.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Data mesh implementation using Hive views"
+author: "Alejandro Rojas, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/Comcast.png
+show_pagenav: false
+---
+
+At Comcast, data is used in a data mesh ecosystem, with a vision where users can
+discover data and request data through self-service. With federation, various
+tools, and the ability to create, read, and write data with different platforms,
+it's a full-blown data mesh. So how do you build that? With Trino, of course,
+and with the power of Hive views. Tune into the 10-minute lightning talk that
+Alejandro gave at Trino Fest to learn more about how Comcast pulled it off.
+
+<!--more-->
+
+{% youtube ZgcVtPFkKHM %}
+
+## Recap
+
+With various different storage systems, like S3, MinIO, and Glue, and users that
+want to be able to use a variety of data platforms, including Trino, but also
+Databricks and Spark, Comcast needed something to sit between the data and those
```

Comment:
whole sentence feels like a bit much. might sound better if it's 2 sentences

Reply:
Not really sure how to break it up tbh

Reply:
Good enough.